src/pubtools/sign/signers/msgsigner.py

```diff
@@ -155,6 +156,7 @@ def _construct_signing_message(
             "request_id": str(uuid.uuid4()),
             "created": isodate_now(),
             "requested_by": self.creator,
+            "repository": repository,
```

Comment:
Radas tries to get 'repo' instead of 'repository' from message.

Reply:
renamed